extensions/pointcloud/README.md

```diff
@@ -0,0 +1,53 @@
+# Point Cloud Extension Specification (`pc`)
+
+**Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
+
+This document explains the fields of the Point Cloud Extension to a STAC Item,
+which allows STAC to more fully describe point cloud datasets. The point clouds can
+come from either active or passive sensors, and data is frequently acquired using
+tools such as LiDAR or coincidence-matched imagery.
+
+- [Example](example-autzen.json)
+- JSON Schema is missing. PRs are welcome.
+
+## Item Fields
+
+| Field Name    | Type                | Description |
+| ------------- | ------------------- | ----------- |
+| pc:count      | integer             | The number of points in the item. |
+| pc:density    | number              | Number of points per square unit area. |
+| pc:encoding   | string              | Content encoding or format of the data. |
+| pc:schema     | [Schema Object]     | A sequential array of items that define the dimensions and their types. |
+| pc:statistics | [Statistics Object] | A sequential array of items mapping to `pc:schema` defines per-channel statistics. |
+| pc:type       | string              | Phenomenology type for the point cloud. Possible valid values might include `lidar`, `eopc`, `radar`, `sonar`. |
+
+### Schema Object
+
+A sequential array of items that define the dimensions and their types. All fields are
+required.
+
+| Field Name | Type    | Description |
+| ---------- | ------- | -------------------------- |
+| name       | string  | The name of the dimension. |
+| size       | integer | The size of the dimension in bytes. Whole bytes only are supported.|
+| type       | string  | Dimension type. Valid values include `floating`, `unsigned`, and `signed`|
```

Comment:
As above, it is potentially a bit easier to understand as follows: "Possible valid values might include `floating`, `unsigned`, and `signed`." (Or are these values exclusive?)

Reply:
They're exclusive, and a dimension can only have a single type of one of those three enumerations.

Reply:
Great, thanks. Resolving this.